.github/ISSUE_TEMPLATE/report_bug.yml

```
@@ -0,0 +1,114 @@
+            </picture>
+          </a>
+        </p>
+  - type: markdown
+    attributes:
+      value: >
+        Thank you for contributing to improve Dagster.
+        If you have a change ready to submit, you do not need to create an issue.
+        Instead, you can open a [pull request](https://github.com/dagster-io/dagster/pulls).
+  - type: input
+    attributes:
+      label: Dagster version
+      description: What Dagster version are you using?
+      placeholder: "Copy and paste the output from `dagster --version`."
+    validations:
+      required: true
+  - type: input
+    attributes:
+      label: Dagit version
+      description: What Dagit version are you using?
+      placeholder: "Copy and paste the output from `dagit --version`."
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Summary
+      description: Provide a brief description of the issue.
+      placeholder: >
+        Please provide a brief description of the issue and the context in which it happened.
+        When possible, add error logs.
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Dagit screenshots
+      description: Provide Dagit screenshots of the issue.
+      placeholder: When possible, provide screenshots or video links that display the issue.
+  - type: textarea
+    attributes:
+      label: What did you expect to happen
+      description: Describe what you think went wrong.
+      placeholder: Please explain why you think the existing behavior is incorrect.
+  - type: textarea
+    attributes:
+      label: How to reproduce
+      description: How can we reproduce the issue?
+      placeholder: Please provide a reproducible guide of how to create the issue.
+  - type: dropdown
+    attributes:
+      label: Deployment type
+      description: What kind of Dagster deployment do you have?
+      multiple: false
+      options:
+        - "Dagster Helm chart"
+        - "Docker Compose"
+        - "Other Docker-based deployment"
+        - "Local"
+        - "Dagster Cloud"
+        - "Other"
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Deployment details
+      description: Add information about your deployment.
+      placeholder: >
+        Enter any relevant details about your deployment. For example, include the version of your
+        tools (docker, helm, kubernetes, etc.) and any customizations or configuration that
+        you added.
+  - type: textarea
+    attributes:
+      label: Additional information
+      description: Anything else we need to know?
+      placeholder: Enter additional details that may be relevant.
+  - type: checkboxes
+    attributes:
+      label: Are you willing to submit PR?
```

I made this comment on the docs template too, but:

```suggestion
      label: Are you willing to submit a PR?
```